modules/standard/List.chpl

```diff
@@ -766,18 +755,30 @@ module List {
     }
 
     /*
-      Returns a reference to the last item in this list.
+      Returns a reference to the first item in this list.
 
       .. warning::
 
         Calling this method on an empty list will cause the currently running
         program to halt. If the `--fast` flag is used, no safety checks will
         be performed.
 
-      :return: A reference to the last item in this list.
+      :return: A reference to the first item in this list.
       :rtype: `ref eltType`
     */
-    proc ref last() ref {
+    proc ref first() ref{
+      ref result = _firstHelper();
+      return result;
+    }
+
+    pragma "no doc"
+    proc const ref first() const ref{
```

Comment:
Nit: Put a space between `ref` and `{`, just to stay consistent with formatting in this module.